CMakeLists.txt

```diff
@@ -39,6 +39,16 @@ message(STATUS "Using CMake version ${CMAKE_VERSION}")
 option(MZ_COMPAT "Enables compatibility layer" ON)
 # Compression library options
 option(MZ_ZLIB "Enables ZLIB compression" ON)
+
+# Controls to select either zlib-ng or zlib
+set(MZ_ZLIB_FLAVOR "auto" CACHE STRING "Select the preferred zlib flavor - auto searches for zlib-ng then zlib")
```

Comment:
Is it possible to use option() here?

Reply:
yes, updating the readme for that is a good idea

Reply:
> Is it possible to use option() here?

option is boolean - it could be framed as `MZ_ZLIB_PREFER_ZLIB-NG` or something like that.
Not sure if that's any better but i can do that if you provide the name 😅

With MZ_SANITIZER we already do it with a string "option".

Reply:
> Should the README be updated to include the new option?

Done